src/centralserver/admin-service/src/main/resources/openapi-definition.yaml

```diff
@@ -2092,6 +2091,42 @@ paths:
       summary: delete a subsystem
       tags:
         - subsystems
+  /members/{id}/subsystems:
+    get:
+      description: <h3>CS Administrator lists subsystems.</h3>
+      operationId: getSubsystems
+      parameters:
+        - description: member code of X-Road member
```

Comment:
Only member_code is not a uniq identifier for the member. The {id} should be the full identifier(instance:class:code), like in other /members/{id} endpoint operations